client/verta/verta/endpoint/build/_build.py

```diff
@@ -102,3 +102,30 @@ def get_scan(self) -> _build_scan.BuildScan:
 
         """
         return _build_scan.BuildScan._get(self._conn, self.id)
+
+    def start_scan(self, external: bool = False) -> _build_scan.BuildScan:
+        """Start a new scan for this build. Internal scans are not yet supported.
+        Use ``external=True`` parameter.
```

Comment:
I'd find it very unpleasant for the default behavior of a function be to encounter an error, but that might just be me 🤔

Reply:
I agree, but at least it's an easy solution handed to you at the moment.  The alternative could be that a working script breaks because a new version of the client comes out and your formerly external-by-default scans are now treated as internal because of a missing parameter.  No great options, but at least this could be temporary and force users into a future-proof state right from the introduction of the feature.

Reply:
Couldn't we just remove the default for now?

```suggestion
    def start_scan(self, external: bool) -> _build_scan.BuildScan:
        """Start a new scan for this build. Internal scans are not yet supported.
        Use ``external=True`` parameter.
```

So a user has to explicitly understand what the parameter does and enter `False`, rather than someone autocompleting thinking it'll work then encountering the `NotImplementedError`.

Reply:
Couldn't we just remove the default for now?

```suggestion
    def start_scan(self, external: bool) -> _build_scan.BuildScan:
        """Start a new scan for this build. Internal scans are not yet supported.
        Use ``external=True`` parameter.
```

So a user has to explicitly understand what the parameter does and enter `False`, rather than someone autocompleting thinking it'll work then encountering the `NotImplementedError`.

Reply:
Yeah, I like that argument.  committing suggestion now.